### ISSUE TITLE
Consider `llms.md`, instead

### ISSUE THREAD

**Issue by kentyman23** [AnswerDotAI/llms-txt#4]
Since it's Markdown, consider using `llms.md`, instead. This will hint to viewers to syntax highlight, allow rendered previews, etc.

**Comment by motdotla** [comment]
yeah, wish this was llms.md

**Comment by ethanresnick** [comment]
`/.well-known/llms.md` would 100% have been the standards-compatible way to do this

**Comment by gregw** [comment]
Also it should be case-insensitive, so LLMs.md works as well

**Comment by david-a-wheeler** [comment]
No, it should be case-sensitive. Many (probably most) web servers are case-sensitive when serving files, so whatever case is stated *is* the one permitted case.

**Comment by romainmenke** [comment]
Definitely should be `/.well-known/llms.md`.
We are already seeing adoption of `llms.txt` so best to make this change quickly
